tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml

```diff
@@ -78,18 +78,28 @@
         </test>
     </tests>
     <help>
+
+.. class:: warningmark
+
+The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
+automatically be converted to binary before they are analyzed.  This step is performed using the
+ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
+**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
+first manually convert the image to binary using the **Convert to binary (black and white) with
```

Comment:
Thanks Bjorn,  I’m starting to get more understanding of this process.  I’ve been building images and seen how some do not always download everything completely, so that may be one of the causes of the issue I was seeing.  Have a great trip back home.  I’ll be in touch!

Greg


> On Nov 21, 2015, at 3:23 PM, Björn Grüning <notifications@github.com <mailto:notifications@github.com>> wrote:
> 
> In tools/image_processing/imagej2/imagej2_analyze_skeleton/imagej2_analyze_skeleton.xml <https://github.com/bgruening/galaxytools/pull/204#discussion_r45550196>:
> 
> > @@ -78,18 +78,28 @@
> >          </test>
> >      </tests>
> >      <help>
> > +
> > +.. class:: warningmark
> > +
> > +The underlying AnalyzeSkeleton Fiji plugin works on binary images, so other image types will
> > +automatically be converted to binary before they are analyzed.  This step is performed using the
> > +ImageJ2 **Make Binary** command with the following settings: **Iterations:** 1, **Count:** 1,
> > +**Black background:** No, **Pad edges when eroding:** No.  If these settings are not appropriate,
> > +first manually convert the image to binary using the **Convert to binary (black and white) with
> I Greg! I'm in NY at the airport currently, on my way back :(
> To you problem, try to rebuild please your image if you are based on :dev.
> @afgane <https://github.com/afgane> have worked a lot on this image during the last 3 days and fixed yesterday this bug: bgruening/docker-galaxy-stable#106 <https://github.com/bgruening/docker-galaxy-stable/pull/106>
> The idea is to test all images automatically with travis :)
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/bgruening/galaxytools/pull/204/files#r45550196>.
>